templates/clientFacade.gotmpl

```diff
@@ -77,12 +77,20 @@ func New(transport runtime.ClientTransport, cfg *TransportConfig, formats strfmt
   cli := new({{ pascalize .Name }})
   cli.cfg = cfg
   cli.Transport = transport
+  cli.formats = formats
```

Comment:
How is this and the change in line 162 related with the cloning?

Reply:
By saving the formats, we call New with the same settings as when it was first called

Reply:
I see, now it's clearer. Thank you!